src/pal/inc/pal.h

```diff
@@ -582,12 +582,12 @@ BOOL
 PALAPI
 PAL_NotifyRuntimeStarted();
 
+static const int MaxPipeNameLength = 64;
```

Comment:
Since this constant is now publicly exposed, could you please name it so that it is clear it belongs to the debugger pipes?